zipfs-law/bin/plotcounts_branch_reject.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import sys
+import re
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+
+def main(args):
+    """Run the command line program."""
+
+    input_csv = args.infile if args.infile else sys.stdin
+    df = pd.read_csv(input_csv, header=None, names=('word', 'word frequency'))
+    df['rank'] = df['word frequency'].rank(ascending=False)
+    df['1/rank'] = 1 / df['rank'] 
```

Comment:
maybe `inv_rank` instead of `1/rank` as a name unless we're planning to show people later that using special characters like `/` in column names makes things difficult? I.e., you can't use `df.1/rank` but you can use `df.inv_rank`.

Reply:
Good suggestion.